its/projects/ExcludedTest/Normal/Program.cs

```diff
@@ -13,17 +13,17 @@ static void Main(string[] args)
 
         }
 
-        public string Bar
+        public static string Bar
```

Comment:
Changed the .Net framework and language version and now these new cases were found (static property and method). I've preferred to fix them to keep the test code as it is.